common/pom.xml

```diff
@@ -97,6 +97,11 @@
             <artifactId>spring-context-support</artifactId>
             <version>4.3.2.RELEASE</version>
         </dependency>
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-test</artifactId>
+            <version>4.3.2.RELEASE</version>
```

Comment:
I think this should have a `<scope>test</scope>` since it is only needed for testing.